iam_policy/variables.tf

```diff
@@ -30,3 +30,20 @@ variable "policy_json" {
     }))
   })
 }
+
+variable "user_tags" {
```

Comment:
We talked about this yesterday in the demo. Naveen asked on what the type should be for `user_tags` & `octopus_tags`. We decided on `map(string)`

Reply:
changed type to map(string)